docs/casts/datalad_convenience_vs_git.sh

```diff
@@ -0,0 +1,109 @@
+say "Fast forward through some prior work..."
+
+say "Let's assume a student developed an algorithm, using Git."
+run "mkdir code; git -C code init; touch code/work; git -C code add work ; git -C code commit -m 'MSc thesis by student A done'"
```

Comment:
To be realistic, drop use of -C

Reply:
That is only the setup. The key bits are the "PI" work.

Reply:
[ a not relevant quip ]

> To be realistic,

It amused me to see this phrase near a snippet where the "algorithm" was an empty file.